### DISCUSSION TITLE
Can't update Netbox docker

### DISCUSSION THREAD

**Discussion by Scheppers-wetteren** [netbox-community/netbox#13928] [Help Wanted!]
I'm trying to update Netbox to version 3.6.3 (from 3.4), but I am unable to do so. I'm getting the following error;

ERROR: The Compose file './docker-compose.yml' is invalid because:
services.netbox-housekeeping.depends_on contains an invalid type, it should be an array
services.netbox-worker.depends_on contains an invalid type, it should be an array

**Comment by arthanson** [comment]
This looks like from netbox-docker, you might get better response opening the issue on the discussion forum there: https://github.com/netbox-community/netbox-docker/discussions
